go/badges/badgestate.go

```diff
@@ -452,24 +451,23 @@ func (b *BadgeState) Clear() {
 	b.walletUnreadMap = make(map[stellar1.AccountID]int)
 }
 
-func (b *BadgeState) updateWithChat(ctx context.Context, update chat1.UnreadUpdate) {
+func (b *BadgeState) updateWithChat(ctx context.Context, update chat1.UnreadUpdate, isMobile bool) {
 	b.log.CDebugf(ctx, "updateWithChat: %s", update)
+	deviceType := keybase1.DeviceType_DESKTOP
```

Comment:
why not just `b.G().IsMobileAppType())` here instead of threading `isMobile`?

Reply:
No `G` here, just thought this was easier.